Sources/Core/Client.swift

```diff
@@ -595,12 +590,16 @@ public final class Client {
         self.watchLoopTask?.cancel()
         self.watchLoopTask = nil
 
```

Comment:
I suspect the following:
``onStreamDisconnect`` runs in ``Task``, so ``self.watchLoopTask`` and ``self.watchLoopReconnectTimer`` can be accessed a thread on which the ``Task`` is running and a thread on which the ``Client`` is running concurrently.

Reply:
<img width="670" alt="스크린샷 2022-11-10 오후 12 43 37" src="https://user-images.githubusercontent.com/12286308/200995752-ee7bb2ef-0a37-49e3-8e67-677a1e00023e.png">